src/ReachSets/ContinuousPost/GLGM06/reach.jl

```diff
@@ -0,0 +1,76 @@
+# ===============================================================
+# Homogeneous case
+# ===============================================================
+function reach_homog!(HR::Vector{ReachSet{Zonotope, Float64}},
+                      Ω0::Zonotope,
+                      Φ::AbstractMatrix,
+                      N::Int,
+                      δ::Float64,
+                      max_order::Int)
+
+    # save timestamps with the reach set
+    t0, t1 = zero(δ), δ
+
+    # initial reach set
+    HR[1] = ReachSet{Zonotope, Float64}(Ω0, t0, t1)
+
+    k = 1
+    while k < N
+        HR_next = linear_map(Φ, HR[k].X)
+        if order(HR_next) > max_order
+            HR_next = reduce_order(HR_next, max_order)
```

Comment:
Maybe it is actually faster to call `reduce_order` and let the condition there decide. `order` will return a `Rational` and you need to convert between numbers. But I understand if you want to keep it the way it is.
(same for the other method below)

Reply:
Ok, and just keep the call to `reduce_order` in all cases.